cmd/swarm/main.go

```diff
@@ -325,6 +327,7 @@ DEPRECATED: use 'swarm db clean'.
 		CorsStringFlag,
 		EnsAPIFlag,
 		EnsAddrFlag,
+		SwarmTomlConfigPathFlag,
```

Comment:
wrong rename: `cmd/swarm/main.go:330:3: undefined: SwarmTomlConfigPathFlag`. This should refer to `swarmTomlConfigFileFlag` I guess.

Reply:
Something went terribly wrong after rebasing. Hang on